src/main/java/org/folio/inventory/resources/AbstractInventoryResource.java

```diff
@@ -0,0 +1,61 @@
+package org.folio.inventory.resources;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.inventory.support.EndpointFailureHandler.getKnownException;
+import static org.folio.inventory.support.EndpointFailureHandler.handleFailure;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.inventory.common.WebContext;
+import org.folio.inventory.storage.Storage;
+import org.folio.inventory.storage.external.Clients;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpClient;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+
+public abstract class AbstractInventoryResource {
```

Comment:
Why does this need to be a abstract class?

Is the intent to re-use the handle method? If so, could it instead be included in it's own class, maybe something like `DefaultHandler`?

Reply:
It is an attempt to generalize inventory resources. Most of them don't have a base class.

Reply:
Ok, let's leave this in place. I'll close this as resolved.

(I'm aware that I am actively biased against inheritance)